README.md

```diff
@@ -165,6 +165,72 @@ lagoon runtime configuration.
 lagoon add variable -p [project name] -e dev -N NETLIFY_SITE_ID -V [netlify site id]
 ```
 
+### Publisher authentication with Drupal
+
+Publisher can require to authenticate with Drupal based on OAuth2.
+It is only used on Lagoon environments.
+
+<details>
+  <summary>How it works</summary>
+  
+  #### Drupal configuration
+  
+  ##### Create keys
+  
+  Per environment, keys are gitignored and are auto-generated via a Lagoon post-rollout task.
+  
+  To generate keys manually
+  
+  via Drush: cd in the cms directory then
+  
+  ```bash
+  drush simple-oauth:generate-keys ./keys
+  ```
+  
+  or via the UI
+  
+  - Go to `/admin/config/people/simple_oauth`
+  - Click on "Generate keys", the directory should be set to `../keys`
```

Comment:
I think we changed this to private files?

Reply:
Thanks, fixed https://github.com/AmazeeLabs/silverback-template/pull/86/commits/b47b2c45ae2782d108e0b57266bb5fa29c4be314